variables.py

```diff
@@ -18,9 +18,9 @@
 
 print("\n------------------- Challenge 1 -------------------\n")
 
-#print(f"Hi! My name is {name}")
-#print(f"I'm in the {grade}th grade.")
-#print(f"A fun fact about me is that {funFact}")
+print(f"Hi! My name is {Nour}")
```

Comment:
In this challenge we would like you to create variables that will get changed through the code.
```suggestion
name = Nour
print(f"Hi! My name is {name}")
```